cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -187,7 +166,9 @@ Slice::Python::getMetaType(const SyntaxTreeBasePtr& p)
     {
         auto contained = dynamic_pointer_cast<Contained>(p);
         assert(contained);
-        string s = "_" + getMappedPackage(contained, '_') + contained->mappedName();
+        string s = "_" + contained->mappedScoped("_");
+        // Replace here is required in case any module is remap with python:identifier into a nested package.
+        replace(s.begin(), s.end(), '.', '_');
```

Comment:
Here we need that '.' in `python:identifier` is also replaced, which is not done by `mappedScoped`.